### DISCUSSION TITLE
No version dropdown or it doesn't have versions?

### DISCUSSION THREAD

**Discussion by stumpylog** [jimporter/mike#221] [Q&A] [answered]
I'm trying to setup mike for versioning a little project and am encoutnering some issues getting the version dropdown to work.

When not hovering over the link, there is an arrow there, but when I actually attempt to select a different version, the arrow disappears and there is no dropdown at all.

The last deploy command run was `mike deploy --push --branch gh-pages --remote origin --update-aliases latest 0.6.0`.  There should be at least 2 versions, 0.5.0 and 0.6.0.  I can manually change `latest` to one of these, but the resulting page is the same.

Clearly I'm doing something wrong, but I'm unsure what.  Any help would be appreciated.

Docs Link: https://stumpylog.github.io/gotenberg-client/latest/

**Comment by jimporter** [comment, marked as answer]
> `mike deploy --push --branch gh-pages --remote origin --update-aliases latest 0.6.0`

You've specified your versions backwards here. The primary version goes first, and then any aliases go after that. Here, you've actually created a version named `latest` with an alias of `0.6.0`. In some other invocation, you've likely done the same but for 0.5 so now you have one version named `latest` with two aliases: https://github.com/stumpylog/gotenberg-client/blob/gh-pages/versions.json

**Reply by stumpylog** [reply]
Thank you, that's what I needed.  Working much better now.
